apps/microtvm/zephyr/template_project/boards.json

```diff
@@ -43,15 +43,15 @@
     },
     "nucleo_f746zg": {
         "board": "nucleo_f746zg",
-        "model": "stm32f746xx",
+        "model": "stm32f7xx",
```

Comment:
why these changes?

Reply:
`stm32f7xx` is more generic than `stm32f746xx`
https://www.st.com/en/microcontrollers-microprocessors/stm32f7-series.html